api-tests/src/steps/ipv-steps.ts

```diff
@@ -178,8 +194,15 @@ Then("I get an OAuth response", function (this: World): void {
 });
 
 When(
-  "I use the OAuth response to get my identity",
-  async function (this: World): Promise<void> {
+  /^I use the OAuth response to get my '(identity|MFA reset result)'$/,
```

Comment:
Do you really want quotes round identity? I don't think it needs them...

IMO it should always read as though it's a real sentence, compare:
- `I use the OAuth response to get my identity` - a legit sentence
- `I submit kenneth-valid details to the CRI` - a bit weird, we want quotes around 'kenneth-valid' as it isn't a real word, it's a reference to the stub data filename